downstream/modules/platform/ref-controller-performance-troubleshooting.adoc

```diff
@@ -0,0 +1,35 @@
+** *Lack of either control or execution capacity*: see “awx_instance_remaining_capacity” metric from the application metrics available on /api/v2/metrics. See xref:con-controller-monitor-controller[Monitoring {ControllerName}] for more information about how to monitor metrics. See xref:ref-controller-capacity-planning[Capacity planning for deploying {ControllerName}] for information on how to plan your deployment to handle the number of jobs you are interested in.
+
+*Jobs run more slowly on {ControllerName} than on a local machine.*
+
+* Some additional overhead is expected, because {ControllerName} may be dispatching your job to a separate node. In this case, {ControllerName} is starting a container and running ansible-playbook there, serializing all output and writing it to a database. 
+* Project update on launch and inventory update on launch behavior can cause additional delays at job start time.
+* Size of projects can impact how long it takes to start the job, as the project is updated on the control node and transferred to the execution node.
+Internal cluster routing can impact network performance. For more information, see xref:ref-controller-internal-cluster-routing[Internal cluster routing].
+* Container pull settings can impact job start time. The execution environment is a container that is used to run jobs within it. Container pull settings can be set to “Always”, “Never” or “If not present”. If the container is always pulled, this can cause delays.
+* Ensure that all cluster nodes, including execution, control, and the database, have been deployed in instances with storage rated to the minimum required IOPS, because the manner in which {ControllerName} runs ansible and caches event data implicates significant disk I/O. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/red_hat_ansible_automation_platform_planning_guide/platform-system-requirements#red_hat_ansible_automation_platform_system_requirements[Red Hat Ansible Automation Platform system requirements].
+
+*Database storage does not stop growing.*
+
+* {ControllerNameStart} has a management job titled “Cleanup Job Details”. By default, it is set to keep 120 days of data and to run once a week. Job data can be voluminous. To reduce the amount of data in the database, you can shorten the retention time. For more information, see xref:proc-controller-remove-old-activity-stream[Removing Old Activity Stream Data].
+* Running the cleanup job will delete the data in the database. However, it is necessary that the database at some point performs its vacuuming operation which reclaims storage. See xref:ref-controller-database-settings[PostgreSQL database configuration and maintenance for {ControllerName}] for more information about database vacuuming.
```

Comment:
Running the cleanup job deletes the data....

However, the database must at some point perform....